examples/basic/play-server/src/main/scala/sample/Server.scala

```diff
@@ -25,8 +26,13 @@ object Server extends App with Results {
     }
     case GET(p"/assets/sample-client-fastopt.js") =>
       controllers.Assets.versioned("/", "sample-client-fastopt.js")
+    case GET(p"/api/description") => Action {
```

Comment:
This endpoint shows a JSON description of the `DocumentedApi` (defined in the file below). This JSON description follows the OpenAPI 3 specification.